### PR TITLE
Replace a few string operator+ by absl::StrCat().

### PR DESCRIPTION
**Pull request by MichaelGrupp**
...in some places that can be called frequently.

### CODE REVIEW

File: cartographer_ros/cartographer_ros/map_builder_bridge.cc [hunk 1]
```diff
@@ -45,7 +46,7 @@ ::std_msgs::ColorRGBA ToMessage(const cartographer::io::FloatColor& color) {
 visualization_msgs::Marker CreateTrajectoryMarker(const int trajectory_id,
                                                   const std::string& frame_id) {
   visualization_msgs::Marker marker;
-  marker.ns = "Trajectory " + std::to_string(trajectory_id);
+  marker.ns = absl::StrCat("Trajectory ", std::to_string(trajectory_id));
```
Comment (sebastianklose) [inline cartographer_ros/cartographer_ros/map_builder_bridge.cc]: you can remove the std::to_string(...) absl::StrCat handles this internally.

File: cartographer_ros/cartographer_ros/node.cc [hunk 2]
```diff
@@ -471,8 +472,9 @@ cartographer_ros_msgs::StatusResponse Node::FinishTrajectoryUnderLock(
 
   cartographer_ros_msgs::StatusResponse status_response;
   if (trajectories_scheduled_for_finish_.count(trajectory_id)) {
-    const std::string message = "Trajectory " + std::to_string(trajectory_id) +
-                                " already pending to finish.";
+    const std::string message =
+        absl::StrCat("Trajectory ", std::to_string(trajectory_id),
```
Comment (sebastianklose) [inline cartographer_ros/cartographer_ros/node.cc]: same here, please remove std::to_string, here and everywhere below.

File: cartographer_ros/cartographer_ros/map_builder_bridge.cc (hunk 1 above)
Comment (MichaelGrupp) [reply cartographer_ros/cartographer_ros/map_builder_bridge.cc]: Cool, didn't know. Will change

**Comment by sebastianklose** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user sebastianklose. Merge queue now has a length of 1.

**Pull request merged**